pupepat/quiver.py

```diff
@@ -39,6 +40,11 @@ def make_quiver_plot(output_dir, output_table, output_plot='pupe-pat'):
         neighbor_inliers = get_neighbor_inliers(focus_set)
         focus_set = focus_set[neighbor_inliers]
 
+        # If there are less than 5 images taken, don't bother making a quiver plot. The quiver plot requires
+        # 2x2 free parameters so the fit will fail for less than 5.
+        if len(focus_set) < 5:
```

Comment:
Could we log something to the console to explain why we didn't create a quiver plot?

Reply:
Better?